trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -167,7 +167,7 @@ This section uses the Augmented Backus-Naur Form (ABNF) notation of [RFC5234](ht
 
 The `OWS` rule defines an optional whitespace. It is used where zero or more whitespace characters might appear. When it is preferred to improve readability - a sender SHOULD generate the optional whitespace as a single space; otherwise, a sender SHOULD NOT generate optional whitespace. See details in corresponding RFC.
 
-The `tracestate` field value is a `list` as defined below. The `list` is a series of `list-members` separated by commas `,`, and a `list-member` is a key/value pair separated by an equals sign `=`. Spaces and horizontal tabs surrounding `list-member`s are ignored. There can be a maximum of 128 `list-member`s in a `list`.
+The `tracestate` field value is a `list` as defined below. The `list` is a series of `list-members` separated by commas `,`, and a `list-member` is a key/value pair separated by an equals sign `=`. Spaces and horizontal tabs surrounding `list-member`s are ignored. There can be a maximum of 32 `list-member`s in a `list`.
```

Comment:
Also update BNF rule below: `list  = list-member 0*127( OWS "," OWS list-member )` to `list  = list-member 0*31( OWS "," OWS list-member )`

Reply:
Done.